targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_WIFI_BT/GeneratedSource/cycfg_qspi_memslot.c

```diff
@@ -231,23 +231,23 @@ const cy_stc_smif_mem_config_t S25FL512S_SlaveSlot_0 =
     /* Determines the slot number where the memory device is placed. */
     .slaveSelect = CY_SMIF_SLAVE_SELECT_0,
     /* Flags. */
-    .flags = CY_SMIF_FLAG_WR_EN,
+    .flags = CY_SMIF_FLAG_MEMORY_MAPPED | CY_SMIF_FLAG_WR_EN,
     /* The data-line selection options for a slave device. */
     .dataSelect = CY_SMIF_DATA_SEL0,
     /* The base address the memory slave is mapped to in the PSoC memory map.
     Valid when the memory-mapped mode is enabled. */
     .baseAddress = 0x18000000U,
     /* The size allocated in the PSoC memory map, for the memory slave device.
     The size is allocated from the base address. Valid when the memory mapped mode is enabled. */
-    .memMappedSize = 0x10000U,
```

Comment:
This was the QSPIF size used in XIP mode. The original 64K was too small to fit a larger cloud client application. I made it 4x bigger in order to fit the code. Since I'm using contiguous memory, this works with the cypress programmer.